src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -0,0 +1,94 @@
+package nextstep.subway.line.application;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import lombok.RequiredArgsConstructor;
+import nextstep.subway.common.domain.model.exception.EntityNotFoundException;
+import nextstep.subway.common.domain.model.exception.FieldDuplicateException;
+import nextstep.subway.line.domain.dto.LineRequest;
+import nextstep.subway.line.domain.dto.LineResponse;
+import nextstep.subway.line.domain.dto.SectionRequest;
+import nextstep.subway.line.domain.model.Line;
+import nextstep.subway.line.domain.repository.LineRepository;
+import nextstep.subway.station.application.StationService;
+import nextstep.subway.station.domain.model.Station;
+
+@RequiredArgsConstructor
+@Service
+@Transactional
+public class LineService {
+    private static final String ENTITY_NAME_FOR_EXCEPTION = "지하철 노선";
+
+    private final LineRepository lineRepository;
+    private final StationService stationService;
+
+    public Line findByIdOrThrow(Long id) {
```

Comment:
```suggestion
    public Line findById(Long id) {
```

메서드 내부 동작을 메서드명에 다 써줄 필요는 없습니다.